src/main/java/bisq/core/dao/governance/voteresult/VoteResultService.java

```diff
@@ -569,10 +572,14 @@ private void applyParamChange(Set<EvaluatedProposal> acceptedEvaluatedProposals,
             if (list.size() == 1) {
                 applyAcceptedChangeParamProposal((ChangeParamProposal) list.get(0).getProposal(), chainHeight);
             } else if (list.size() > 1) {
+                log.warn("There have been multiple winning param change proposals with the same item. " +
+                        "This is a sign of a social consensus failure. " +
+                        "We treat all requests as failed in such a case.");
```

Comment:
I think this is correct, with a social failure it's unsafe to continue. I don't think this is an attack vector since it needs more than 50% of votes attacking, which is already a bigger issue in itself.